pkg/kwok/cmd/root.go

```diff
@@ -65,10 +65,11 @@ func NewCommand(logger logger.Logger) *cobra.Command {
 	)
 
 	cmd := &cobra.Command{
-		Args:  cobra.NoArgs,
-		Use:   "kwok [command]",
-		Short: "kwok is a tool for simulate thousands of fake kubelets",
-		Long:  "kwok is a tool for simulate thousands of fake kubelets",
+		Args:         cobra.NoArgs,
+		Use:          "kwok [command]",
+		Short:        "kwok is a tool for simulate thousands of fake kubelets",
+		Long:         "kwok is a tool for simulate thousands of fake kubelets",
+		SilenceUsage: true,
```

Comment:
It seems to have missed some `cobra.Command`s that don't have this option, I counted 15 of them and we need to add this option.

Reply:
for parent command, we can puts out the help for the command instead of print error. 


```

Error: subcommand is required
Usage:
  kwokctl [command] [flags]
  kwokctl [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  create      Creates one of [cluster]
  delete      Deletes one of [cluster]
  get         Gets one of [artifacts, clusters, kubeconfig]
  help        Help about any command
  kubectl     kubectl in cluster
  logs        Logs one of [audit, etcd, kube-apiserver, kube-controller-manager, kube-scheduler, kwok-controller, prometheus]
  snapshot    Snapshot [save, restore] one of cluster

Flags:
  -h, --help          help for kwokctl
      --name string   cluster name (default "kwok")

Use "kwokctl [command] --help" for more information about a command.

exit status 1
```

expected:

```
Kwokctl is a Kwok cluster management tool

Usage:
  kwokctl [command] [flags]
  kwokctl [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  create      Creates one of [cluster]
  delete      Deletes one of [cluster]
  get         Gets one of [artifacts, clusters, kubeconfig]
  help        Help about any command
  kubectl     kubectl in cluster
  logs        Logs one of [audit, etcd, kube-apiserver, kube-controller-manager, kube-scheduler, kwok-controller, prometheus]
  snapshot    Snapshot [save, restore] one of cluster

Flags:
  -h, --help          help for kwokctl
      --name string   cluster name (default "kwok")

Use "kwokctl [command] --help" for more information about a command.
```